db/migrate/20150508140514_create_builds.rb

```diff
@@ -1,10 +1,10 @@
-class CreateBuildArtifact < ActiveRecord::Migration
+class CreateBuilds < ActiveRecord::Migration
   def change
     create_table :builds do |t|
       t.belongs_to :project,    null: false, index: true
-      t.string :git_sha
+      t.string :git_sha,        limit: 128
```

Comment:
sha2 can be up to 128 characters. Probably overkill, but figured it didn't hurt either.

> On May 31, 2015, at 11:08 PM, Roman Sandler <notifications@github.com> wrote:
> 
> In db/migrate/20150508140514_create_builds.rb:
> 
> >    def change
> >      create_table :builds do |t|
> >        t.belongs_to :project,    null: false, index: true
> > -      t.string :git_sha
> > +      t.string :git_sha,        limit: 128
> sha1 is always 40 characters - why pick 128?
> 
> —
> Reply to this email directly or view it on GitHub.
>